Resources/public/js/views/fields/ez-richtext-editview.js

```diff
@@ -293,15 +293,29 @@ YUI.add('ez-richtext-editview', function (Y) {
          */
         _getEditorContent: function () {
             var data = this.get('editor').get('nativeEditor').getData(),
-                section = Y.Node.create(data);
+                root, i, element, list, section,
+                doc = document.createDocumentFragment();
+
+            root = document.createElement('div');
+            doc.appendChild(root);
+            root.innerHTML = data;
+            list = root.querySelectorAll('[id]');
+
+            if (list.length) {
+                for (i = 0; i != list.length; ++i) {
+                    element = list[i];
```

Comment:
you don't really need a local variable, `list[i].removeAttribute("id");` would work well here.